samples/cpp/benchmark_app/main.cpp

```diff
@@ -407,14 +410,14 @@ int main(int argc, char* argv[]) {
             slog::info << "Loading network files" << slog::endl;
 
             auto startTime = Time::now();
-            CNNNetwork cnnNetwork = ie.ReadNetwork(FLAGS_m);
+            auto cnnNetwork = ie.read_model(FLAGS_m);
             auto duration_ms = double_to_string(get_duration_ms_till_now(startTime));
             slog::info << "Read network took " << duration_ms << " ms" << slog::endl;
             if (statistics)
                 statistics->addParameters(StatisticsReport::Category::EXECUTION_RESULTS,
                                           {{"read network time (ms)", duration_ms}});
 
-            const InputsDataMap inputInfo(cnnNetwork.getInputsInfo());
+            const ov::ParameterVector& inputInfo = cnnNetwork->get_parameters();
```

Comment:
```suggestion
            const ov::Output<ov::Node>& inputs = cnnNetwork->inputs();
```

Reply:
Done. whole app is refactored to use inputs instead of get_parameters